lgsf/aws_lambda/handlers.py

```diff
@@ -9,6 +9,22 @@
 from lgsf.path_utils import load_scraper
 
 
+def scraper_worker_handler(event, context):
+    print(f"EVENT: {event}")
```

Comment:
Do we still need this debug?

Reply:
No I guess not - probably better to have an actual log entry from the scraper saying "beginning to scrape 'x' from 'y'"